src/main/resources/application.properties

```diff
@@ -13,4 +13,4 @@ aws.accessKeyId=accessKey
 aws.region=eu-west-2
 aws.sourceBucket=gap-sandbox-attachments
 
-jwt.secretKey=sameSecretAsLambdaAuthorizer
\ No newline at end of file
+jwt.secretKey=${JWT_SECRET_KEY:sameAsLambdaAuthoriser}
```

Comment:
in terraform we overwrite this with what sandbox or qa would need (the same secret key that the lambda authorizer use to generate the jwt token). 
would this change affect that?

Reply:
I put this in for working locally, but yeah i get what you mean, If we have a local env var as JWT_SECRETKEY, would this property be set?